mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -103,3 +133,64 @@ properties:
               A percentage of the cluster's CPU capacity. Can be from 10% to 80%.
               When a cluster's CPU utilization exceeds the target that you have set,
               Bigtable immediately adds nodes to the cluster. When CPU utilization is substantially lower than the target, Bigtable removes nodes. If not set will default to 50%.
+  - !ruby/object:Api::Type::NestedObject
+    name: optimized
+    description:
+      Contains settings for the Optimized store that will be created to serve featureValues for all FeatureViews under this FeatureOnlineStore
+    conflicts:
+      - embeddingManagement
+    exactly_one_of:
+      - bigtable
+      - optimized
+    min_version: beta
+    properties:
+      []  # Meant to be an empty object with no properties - see here : https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.featureOnlineStores#Optimized
+      # The fields below are necessary to include the "Optimized" transformation in the payload
+    send_empty_value: true
+    allow_empty_object: true
+
+  - !ruby/object:Api::Type::NestedObject
+    name: dedicatedServingEndpoint
+    description: |
+       The dedicated serving endpoint for this FeatureOnlineStore, which is different from common vertex service endpoint. Only need to set when you choose Optimized storage type or enable EmbeddingManagement. Will use public endpoint by default.
+    default_from_api: true
+    min_version: beta
+    properties:
+      - !ruby/object:Api::Type::String
+        name: publicEndpointDomainName
+        description: |
+          populated with the domain name to use for this FeatureOnlineStore
```

Comment:
This will also show up in the docs. These should generally be sentences, with at least the first character capitalized. I would personally suggest removing "populated with", since it isn't commonly included in these descriptions, and it is redundant with the "(Output)" prefix that will be generated.

Similar suggestion for some other fields below.

Reply:
Updated!